### PR TITLE
Generate directly callable script in windows

### PR DESCRIPTION
**Pull request by starofrainnight**
After installed retext by pip or setup.py, you must invoke it manually from command line in Scripts directory: "python retext". I think it's better to generate a simple wrapper batch script in Scripts directory, so that ReText will more like a program in windows (at least working just like the other scripts in Scripts directory).

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 import re
 import sys
+import os.path
```
Comment (mitya57) [inline setup.py]: Unused?

File: setup.py [hunk 2]
```diff
@@ -57,8 +58,16 @@ def run(self):
...
+			renamed_file = file[:-3]
+
+			log.info('renaming %s to %s', file, renamed_file)
+			shutil.move(file, renamed_file)
+
+			if sys.platform == "win32":
+				# Generate batch script for warpper the python script so we could invoke that
+				# script directly from command line
+				batch_script = "@echo off\n%s %s %%*" % (sys.executable, renamed_file)
+				open("%s.bat" %  renamed_file, "wb").write(batch_script.encode())
```
Comment (mitya57) [inline setup.py]: Use

```python
with open("%s.bat" %  renamed_file, "w") as bat_file:
    bat_file.write(batch_script)
```

(Context to properly close the file and no binary mode)

**Comment by mitya57** [comment]
Thanks! A couple of comments and I'll merge it.

File: setup.py [hunk 3]
```diff
@@ -57,8 +58,16 @@ def run(self):
...
 		for file in self.get_outputs():
-			log.info('renaming %s to %s', file, file[:-3])
-			shutil.move(file, file[:-3])
+			renamed_file = file[:-3]
+
+			log.info('renaming %s to %s', file, renamed_file)
+			shutil.move(file, renamed_file)
+
+			if sys.platform == "win32":
+				# Generate batch script for warpper the python script so we could invoke that
```
Comment (mitya57) [inline setup.py]: warpper → wrapper

File: setup.py (hunk 1 above)
Comment (starofrainnight) [inline setup.py]: Yep! I forget to remove the test code line.

**Comment by mitya57** [comment]
Thanks! I have squashed your commits into a single one and pushed that.

**Comment by starofrainnight** [comment]
@mitya57 

OK, I will squash my commits before start a PR next time :)

**Pull request closed without merging**